packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -50,6 +50,13 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
         const { OTTER_MODULE_KEYWORD, OTTER_MODULE_SUPPORTED_SCOPES } = await import('@o3r/schematics');
         const displayModuleListRule = displayModuleList(OTTER_MODULE_KEYWORD, OTTER_MODULE_SUPPORTED_SCOPES);
         return () => displayModuleListRule(t, c);
+      },
+      (_, c) => {
+        if (!options.projectName && !options.skipGit && options.commit) {
```

Comment:
`options.projectName` will be automatically filled by `$source` even if not sent (if there is only one project)
are we ok with not trying to setup git if there already is a project ?
it feels weird since we'll do the setup if there are 2 projects

Reply:
The purpose is to commit only when the `ng-add @o3r/core` is apply to the workspace.
In case of 2 projects, if no `projectName` specified, the `ng add` will be applied to the workspace so I would like to commit